theme/cerveza/src/components/_Tooltip.scss

```diff
@@ -0,0 +1,50 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700');
+@import "../utilities";
```

Comment:
I don't think we've switched yet, but instead of import I think we are switching to @use